projects/core/src/features-config/feature-toggles/config/feature-toggles.ts

```diff
@@ -358,6 +358,12 @@ export interface FeatureTogglesInterface {
    * `AddToWishListComponent`, `ProductIntroComponent`, `ProductImageZoomTriggerComponent`
    */
   a11yLinkBtnsToTertiaryBtns?: boolean;
+
+  /**
+   * When enabled, styles for the `cx-bottom-header-slot` class will be applied. These styles are necessary to display
+   * customization buttons in the BottomHeaderSlot in SmartEdit.
+   */
+  enableBottomHeaderSlotStyles?: boolean;
```

Comment:
This feature toggle applies `flex` css styles to cms BottomHeaderSlot.
What do you think about a bit more meaningful toggle name?
`cmsBottomHeaderSlotUsingFlexStyles`